project/src/main/java/com/google/sps/data/PlacesFetcher.java

```diff
@@ -55,20 +60,32 @@
      *     for places or for places details
      */
     public ImmutableList<Place> fetch(UserPreferences preferences) throws FetcherException {
-        TextSearchRequest query =
-            PlacesApi.textSearchQuery(
-                CONTEXT, createCuisinesQuery(preferences.cuisines()), preferences.location())
-                .radius(SEARCH_RADIUS)
+        PlacesSearchResult[] placesSearchResult;
+        int radiusMult = 1;
```

Comment:
Initialize this with 0 and change below to < instead of <=

Reply:
I initialized it to 1 because it multiplies the initial search radius (which if first multiplied by 1). I changed it to be initiated to 0 and updated in the begining of the attempt so I hope now it's clearer.

And you are right about < , changed it.